snuba/datasets/configuration/issues/storages/search_issues.yaml

```diff
@@ -89,6 +89,15 @@ allocation_policies:
         is_enforced: 0
         is_active: 0
 
+delete_allocation_policies:
+  - name: DeleteConcurrentRateLimitAllocationPolicy
+    args:
+      required_tenant_types:
+        - referrer
+        - project_id
```

Comment:
Would the org be also needed or valuable to have?

Reply:
yeah I can add it in